package.json

```diff
@@ -18,13 +18,14 @@
     "tdd": "jest --watch",
     "lint": "yarn prettier-check && yarn eslint",
     "eslint": "eslint --quiet '+(e|web)/**/*.{ts,tsx,js,jsx}'",
-    "type-check": "tsc --noEmit",
+    "type-check": "NODE_OPTIONS='--max-old-space-size=4096' tsc --noEmit",
     "prettier-check": "yarn prettier --check '+(e|web)/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn prettier --write --loglevel silent '+(e|web)/**/*.{ts,tsx,js,jsx}'",
     "nop": "exit 0"
   },
   "private": true,
   "resolutions": {
+    "@swc/core": "1.3.36",
```

Comment:
Why is this needed on v12 and not on master?

Reply:
On master/v13, the dependencies for swc are set in `yarn.lock` to the versions that work. If I don't pin this, we get an incompatible `@swc/core` package and the build errors.

Reply:
I see that master uses @vitejs/plugin-react-swc `^3.2.0` vs `^3.3.1` on this branch. I managed to massage yarn.lock into resolving @swc/core to 1.3.36 (a1bfb9f8c53e58) by doing this:

1. Remove @vitejs/plugin-react-swc from package json.
2. Run `yarn`.
3. Add it back to package.json with `"@vitejs/plugin-react-swc": "~3.2.0"` (this won't upgrade it past 3.2.x).
4. Run `yarn`.
5. Remove @swc/core resolution.

What do you think about about restricting plugin-react-swc to `~3.2.0` instead of adding that @swc/core resolution? I care about this only because adding a resolution is a pretty heavy-handed hammer that I think we should use only as a last resort.